planet/src/main/webapp/js/itinerary.js

```diff
@@ -49,19 +63,67 @@ function renderStartingLocation() {
     }
 }
 
+function renderListOptions() {
```

Comment:
what does 'list' refer to? List of events? List of itineraries?

Reply:
It's the list of events. We're not saving itineraries in the back end